bip-paytoanchor.mediawiki

```diff
@@ -0,0 +1,76 @@
+<pre>
+  BIP: ?
+  Layer: Applications
+  Title: Pay To Anchor (P2A)
+  Author: Gregory Sanders <gsanders87@gmail.com>
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-?
+  Status: Draft
+  Type: Informational
+  Created: 2025-09-23
+  License: BSD-3-Clause
+  Post-History: 2024-06-27: https://github.com/bitcoin/bitcoin/pull/30352 Github PR
+                2024-10-10: https://bitcoinops.org/en/bitcoin-core-28-wallet-integration-guide/ wallet integration guide
+</pre>
+
+==Abstract==
+
+This document describes a new standard output script called PayToAnchor (P2A) and prescribes making spending of this output type standard.
+
+This output is "keyless" meaning no signature or witness data at all are required to spend this output typea.
+
+==Motivation==
+
+The "anchor" output type is a commonly used pattern in Bitcoin layer 2 systems such as Lightning Network and beyond. This pattern allows transactions which are presigned far in advance to leave a "hook" for CPFP fee bumping later, when prevailing feerates for block inclusion are better known.
+
+When the wallet user is unconcerned by *who* is able to bump the transaction fees as long as someone does, this output type may confer a couple of advantages:
+
+1. Using an output type with minimal output and spending input footprint saves fees on average
+2. Keyless operation aids in interoperability of services like watchtowers as there is no privelaged key material for bumping
+
+with the drawback that selection of this output type could result in griefing if anti-pinning measures are not taken.
+
+===Specification===
+
+A P2A output is defined as one with the exact witness output script
```

Comment:
As you're defining a new standard output type, you also need to specify full script verification rules. For instance, due to clean stack rules if you just do things vanilla, you'll end up with two stack elements, which will fail. 

I presume the desired semantics here is to _remove_ the data pushes from the stack?